doc/classes/InputEventJoypadButton.xml

```diff
@@ -4,6 +4,7 @@
 		Input event type for joypad button events.
```

Comment:
Joypads can either refer to game controllers and the 4 directional arrows on a controller. It's also a dated word imo.
`Input event for gamepad buttons.`